src/ixon/meta.rs

```diff
@@ -21,17 +59,18 @@ pub enum ReducibilityHints {
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub enum Metadatum {
-    Name(Name),
-    Info(BinderInfo),
     Link(Address),
+    Info(BinderInfo),
     Hints(ReducibilityHints),
-    All(Vec<Name>),
-    MutCtx(Vec<Vec<Name>>),
+    Links(Vec<Address>),
+    Map(Vec<(Address, Address)>),
+    KVMap(Vec<(Address, DataValue)>),
+    Muts(Vec<Vec<Address>>),
 }
 
 #[derive(Debug, Clone, PartialEq, Eq)]
 pub struct Metadata {
-    pub map: Vec<(Nat, Vec<Metadatum>)>,
+    pub nodes: Vec<Metadatum>,
 }
 
 //impl Serialize for BinderInfo {
```

Comment:
`Serialize` is now implemented in `ixon.rs`, right? I think these commented lines can be deleted.